src/modules/job-exec/testexec.c

```diff
@@ -28,6 +28,10 @@
  *   "wait_status":i        - report this value as status in the "finish" resp
  *   "mock_exception":s     - mock an exception during this phase of job
```

Comment:
commit message for 6d68af3679ae515b6a242ab0c0b:
s/wheck/when/
s/reponse/response/